README.md

```diff
@@ -514,28 +532,27 @@ render({fonts}) {
 The last piece of the puzzle is how do we write the `onFavorite` method.
 
 ```
-onFavorite({eventlet: {value}}) {   
+onFavorite({eventlet: {value}}) {
     value.favorite = true;
 },
 ```
 
 We can also add an `onUnfavorite` method.
 
 ```
-onFavorite({eventlet: {value}}) {   
+onFavorite({eventlet: {value}}) {
```

Comment:
This looks like a typo (present before your PR), and should be `onUnfavorite` I think.